department-of-veterans-affairs/caseflow
# Removing a hearing day

Resolves #7604 

### Description
This PR adds functionality for deleting hearing days when no veterans are scheduled.

### Testing Plan
1) Create a hearing day in Caseflow
2) Navigate to the daily docket
3) Delete the hearing
4) Confirm it works as expected
5) Navigate to a hearing day that has scheduled hearings
6) Confirm the 'Remove Hearing Day' link is not visible
